### PR TITLE
Add docs for for how to run the styleguide

### PR DESCRIPTION
**Pull request by azevedo-252**

### CODE REVIEW

**Comment by naps62** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ Run the setup to get started:
 bin/setup
 ```
 
+Launch our styleguide:
+
+```bash
+rake styleguide
```
Comment (zamith) [inline README.md]: When running `rake` is usually safer to add `bundle exec` before.

**Comment by azevedo-252** [comment]
@zamith bump

**Comment by zamith** [comment]
👍

**Pull request merged**